evap/staff/views.py

```diff
@@ -1828,11 +1828,16 @@ def evaluation_textanswer_edit(request, textanswer_id):
     assert_textanswer_review_permissions(evaluation)
 
     form = TextAnswerForm(request.POST or None, instance=textanswer)
-
+    view = request.GET.get("view", "quick")
```

Comment:
I think it would be better if we did not duplicate the knowledge of what the default view is here; instead, we can just omit adding a `view=...` query string in the redirect here when we were not called with one ourselves. Additionally, I think it would slightly nicer if this would be called "next-view" instead of just "view", because we are not actually doing anything with it here but only after we are done.